test/sai_test/config/port_configer.py

```diff
@@ -442,6 +443,30 @@ def get_mtu(self):
         '''
         return int(self.config.get('mtu'))
 
+    def get_cpu_port_queue(self):
```

Comment:
add comments for this method
move the assignement method to the helper. explicit change the test's attributes outside of the configer(in the help method). then we can reuse this method just for getting the attribute but without changing the test object

Reply:
we can get cpu queue by self.cpuqueue{num}